.github/workflows/ci.yml

```diff
@@ -1,4 +1,4 @@
-name: Test
+name: CI
```

Comment:
Sorry I meant this one, `CI` is a too broad name for this IMO, all workflows are CI, maybe this one should be `Build/Test` or something ?

Reply:
I think now that almost all job run in this workflow the name is appropriate, but I can also change it if you would prefer that.